sql/hive-thriftserver/src/main/java/org/apache/hive/service/cli/session/HiveSessionImpl.java

```diff
@@ -665,6 +667,25 @@ public void close() throws HiveSQLException {
     }
   }
 
+  private void cleanupPipeoutFile() {
+    String lScratchDir = hiveConf.getVar(ConfVars.LOCALSCRATCHDIR);
+    String sessionID = hiveConf.getVar(ConfVars.HIVESESSIONID);
+
+    File[] fileAry = new File(lScratchDir).listFiles(
```

Comment:
Nit: There is an overloaded version of [listfiles](https://docs.oracle.com/javase/8/docs/api/java/io/File.html#listFiles-java.io.FilenameFilter-).
Using it saves 1 line.

Reply:
Sorry i did not find the overloaded version of listfiles which saves 1 line.

Reply:
@attilapiros  will fix it, thanks!